lib/reporters/TestReporter.js

```diff
@@ -0,0 +1,142 @@
+import {Test, Suite} from "../Data.js";
```

Comment:
Agreed. What about the `TestReporter` class? I think it's quite useful to have this in `lib/`: next to testing our own adapters, this can be used to test if an external test framework implements the JsReporters standard correctly. Maybe we should just move the reference data?